lms/templates/hidden_content.html

```diff
@@ -5,22 +5,35 @@
 %>
 
 <div class="sequence hidden-content proctored-exam completed">
-  <h3>
-      ${_("The due date for this {subsection_format} has passed.").format(
-        subsection_format=subsection_format,
-      )}
-  </h3>
-  <hr>
-  <p>
-      ${Text(_(
-          "Because the due date has passed, this {subsection_format} "
-          "is no longer available.{line_break}If you have completed this {subsection_format}, "
-          "your grade is available on the {link_start}progress page{link_end}."
-        )).format(
-          subsection_format=subsection_format,
-          line_break=HTML("<br>"),
-          link_start=HTML("<a href='{}'>").format(progress_url),
-          link_end=HTML("</a>"),
-      )}
-  </p>
+    <h3>
```

Comment:
Looking at this file, I think it was the right call not to break out a separate template.